q2cli/util.py

```diff
@@ -178,3 +182,22 @@ def callback(ctx, param, value):
     return click.Option(['--citations'], is_flag=True, expose_value=False,
                         is_eager=True, callback=callback,
                         help='Show citations and exit.')
+
+
+def usage_example_option(action):
+    import click
+
+    def callback(ctx, param, value):
+        if not value or ctx.resilient_parsing:
+            return
+        else:
+            import q2cli.core.usage as usage
+
+        action = ctx.command._get_action()
+        examples = usage.examples(action)
+        click.secho(examples)
```

Comment:
This should show a message when the action has no registered examples, right now it looks like this:

```bash
(q2dev) 
~/projects/qiime2/q2cli
☻  QIIMETEST= qiime dummy-plugin failing-pipeline --examples                                                                                                                                                                                                                                  cli-usage e0d1766 ✗

(q2dev) 
~/projects/qiime2/q2cli
☻      
```